scripts/src/qc_table_functions.R

```diff
@@ -0,0 +1,374 @@
+options(cli.unicode = FALSE)
+library(tidyverse)
+library(data.table)
+library(magrittr)
+library(data.table)
+library (PRROC)
+
```

Comment:
Currently the return values from the different functions are implicitly defined based on the last object.

I think it would be good to explicitly return the final object you calculated.